tests/UserAgentsTest/LiteTest.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * This file is part of the browscap package.
+ *
+ * Copyright (c) 1998-2017, Browser Capabilities Project
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+namespace UserAgentsTest;
+
+use Browscap\Coverage\Processor;
+use Browscap\Data\PropertyHolder;
+use Browscap\Filter\LiteFilter;
+use Browscap\Formatter\PhpFormatter;
+use Browscap\Generator\BuildGenerator;
+use Browscap\Helper\CollectionCreator;
+use Browscap\Writer\IniWriter;
+use Browscap\Writer\WriterCollection;
+use BrowscapPHP\Browscap;
+use BrowscapPHP\BrowscapUpdater;
+use BrowscapPHP\Formatter\LegacyFormatter;
+use Monolog\Handler\NullHandler;
+use Monolog\Logger;
+use WurflCache\Adapter\File;
+
+/**
+ * @group      useragenttest
+ * @group      lite
+ */
+class LiteTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var \BrowscapPHP\Browscap
+     */
+    private static $browscap = null;
+
+    /**
+     * @var \BrowscapPHP\BrowscapUpdater
+     */
+    private static $browscapUpdater = null;
+
+    /**
+     * @var string
+     */
+    private static $buildFolder = null;
+
+    /**
+     * @var \Browscap\Data\PropertyHolder
+     */
+    private static $propertyHolder = null;
+
+    /**
+     * @var string[]
+     */
+    private static $coveredPatterns = [];
+
+    /**
+     * @var \Browscap\Filter\LiteFilter
+     */
+    private static $filter = null;
+
+    public static function setUpBeforeClass()
+    {
+        // First, generate the INI files
+        $buildNumber    = time();
+        $resourceFolder = __DIR__ . '/../../resources/';
+
+        self::$buildFolder = __DIR__ . '/../../build/browscap-ua-test-' . $buildNumber . '/build/';
```

Comment:
Before these changes, a run that would run the Lite, Standard and Full tests would write to the same directory, since the `$buildNumber` was shared in all runs, with this change, they could potentially write to different directories, is this desired?

If so, would it be helpful to mark the directories differently for each run, i.e. ?

`browscap-ua-test-full-' . $buildNumber . '`
`browscap-ua-test-standard-' . $buildNumber . '`
`browscap-ua-test-lite-' . $buildNumber . '`

Reply:
Folders are named now, like you suggested